Gemfile

```diff
@@ -37,12 +37,13 @@ end
 group :development do
   gem 'capistrano', '~> 3.0'
   gem 'capistrano-bundler', '~> 1.1'
+  gem 'debug'
   gem 'dlss-capistrano', require: false
+  gem "ruby-debug-completion"
 end
 
 group :development, :test do
-  gem 'byebug'
```

Comment:
From Slack it seemed like 'debug' was the future 'byebug' was the past? If you'd rather keep it I can put it back.

Reply:
i lost track and will go with the flow